ui/static/ui/js/learning_resources.jsx

```diff
@@ -379,12 +386,17 @@ define('learning_resources',
         thiz.setState({
           message: {error: "Unable to save form"}
         });
+      }).always(function() {
+        thiz.setState({loaded: true});
```

Comment:
This makes the spinner go away even on failure but I don't think that would trigger future success events. Can you think of a scenario where it would?